requirements.txt

```diff
@@ -1,4 +1,7 @@
 rdflib
 dominate
+scipy
```

Comment:
Do we need nilearn, sklearn and scipy?

Reply:
Scipy is needed as it is a dependency for some of the other modules and is not automatically installed when `setup.py` is run. The others are not, and have been removed.